api/src/db/migrations/v107_refactor_interstate_transport_essential_question.ts

```diff
@@ -0,0 +1,414 @@
+import { v106UserData } from "./v106_add_pet_care_housing_essential_question";
+
+export interface v107UserData {
+  user: v107BusinessUser;
+  profileData: v107ProfileData;
+  formProgress: v107FormProgress;
+  taskProgress: Record<string, v107TaskProgress>;
+  taskItemChecklist: Record<string, boolean>;
+  licenseData: v107LicenseData | undefined;
+  preferences: v107Preferences;
+  taxFilingData: v107TaxFilingData;
+  formationData: v107FormationData;
+  lastUpdatedISO: string | undefined;
+  version: number;
+}
+
+export const migrate_v106_to_v107 = (v106Data: v106UserData): v107UserData => {
+  const getLogisticsResponse = () => {
+    if (
+      v106Data.profileData.industryId === "logistics" &&
+      v106Data.profileData.interstateTransport === true
+    ) {
+      return true;
+    } else if (
+      v106Data.profileData.industryId === "logistics" &&
+      v106Data.profileData.interstateTransport === false
+    ) {
+      return false;
```

Comment:
This if/else-if chain of logic looks correct, but could be simplified quite a lot to make it more readable.

```
if (v106Data.profileData.industryId === "logistics") {
 return ...
} else {
  return ...
}
```

Same with the one below as well